### PR TITLE
install dask lab ext from main

### LINKED ISSUE #1012: [bug] Unable to connect to dashboard using Dask-Gateway and Labextension
## Describe the bug
Running the following on qhub `v0.3.13` and `v0.3.14` generates an accessible link for the dask cluster dashboards, but we are unable to reuse that link to access the dask extension.

```
from dask_gateway import Gateway
gateway = Gateway()

options = gateway.cluster_options()
options

cluster = gateway.new_cluster(options)
cluster
``` 
successful generates a working URL for the dashboards (can be accessed):
image

Dashboard URL does not work:
image

## Expected behavior

- The Dask lab extension should connect to the dask-gateway spawned cluster

## How to reproduce

- Launch a dask-gateway cluster and include the generated Dashboard URI in the Dask extension bar

### PR DESCRIPTION
**Pull request by iameskild**
Fixes | Closes | Resolves #1012 

> Please remove anything marked as optional that you don't need to fill in.
> Choose one of the keywords preceding to refer to the issue this PR solves, followed by the issue number (e.g Fixes # 666).
> If there is no issue, remove the line. Remove this note after reading.

## Changes introduced in this PR:

-

## Types of changes

What types of changes does your PR introduce?

_Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

### Requires testing

- [x] Yes
- [ ] No

### In case you checked yes, did you write tests?

- [ ] Yes
- [ ] No

## Documentation

Does your contribution include breaking changes or deprecations?
If so have you updated the documentation?

- [ ] Yes, docstrings
- [ ] Yes, main documentation
- [ ] Yes, deprecation notices

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered and more.

This updated jupyterlab image has already been added to `beta` deployment if anyone wants to test it out. We needed this for the demon on Monday. 

To enable the extension try:
- `Settings > Advanced settings Editor > Dask`
- There will be a box called "Determine whether to validate dashboards via browser check".
- Just check that and this will update the extension.

@viniciusdc is there a way to automatically enable the extention?

### CODE REVIEW

**Comment by viniciusdc** [comment]
Yes there is https://github.com/Quansight/qhub/issues/1012#issuecomment-1149928436

**Comment by viniciusdc** [comment]
I can have this automatically set by tomorrow, I was hoping to get this by today but lost track of time. Let me know if this is acceptable

File: qhub/template/image/jupyterlab/environment.yaml [hunk 1]
```diff
@@ -54,3 +54,4 @@ dependencies:
  - pip:
      # vscode jupyterlab launcher
      - git+https://github.com/betatim/vscode-binder
+     - git+https://github.com/dask/dask-labextension.git@main
```
Comment (aktech) [inline qhub/template/image/jupyterlab/environment.yaml:57]: Should we rather install it from the commit hash?

Comment (viniciusdc) [reply qhub/template/image/jupyterlab/environment.yaml:57]: I would appreciate that as well, even though the labextension repo does not release frequently, I think it's worth pinning.

Comment (viniciusdc) [inline qhub/template/image/jupyterlab/environment.yaml:57]:
```suggestion
     - git+https://github.com/dask/dask-labextension.git@fc8de03447778e0605d0abafa3d3135f9e49e0d6
```

Comment (iameskild) [reply qhub/template/image/jupyterlab/environment.yaml]: Good idea! Done.

**Review by viniciusdc — APPROVED** [review]
LGTM

**Comment by ian-r-rose** [comment]
Sorry to be slow in publishing, this feature should now be available from pypi

**Pull request merged**